pkgs/applications/editors/bluej/default.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv, fetchurl, makeWrapper, jetbrains }:
+
+stdenv.mkDerivation rec {
+  pname = "bluej";
+  version = "4.2.2";
+  src = fetchurl {
+    url = "https://www.bluej.org/download/files/BlueJ-linux-${builtins.replaceStrings ["."] [""] version}.deb";
+    sha256 = "sha256-XCJB8iCOmPz5qtfHooK88W5v1UP6pf2wuZs00QIxE8M=";
```

Comment:
Have never seen such a sha256. I'm surprised it is compatible. How about:

```
    sha256 = "0000000000000000000000000000000000000000000000000000";
```

Reply:
Probably because I'm using nix unstable. This is what TOFU gave me. I've updated it to the hex version of that base64 string.